src/components/menu/menu.config.ts

```diff
@@ -12,6 +12,9 @@ export default {
     alternate: "var(--colorsComponentsMenuSpringChildAlt500)",
     alternateHover: "var(--colorsComponentsMenuSpringChildAlt600)",
     divider: "var(--colorsComponentsMenuSpringChild400)",
+    scrollbarThumb: "#597a8b",
```

Comment:
question: are we not able to use tokens here?

Reply:
Messaged you privately but just for context for the other reviewer. Harpal advised that because the colours aren't available as a design-token, to use this hex value.